src/seed.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require('mongoose');
+const { Schema, Number } = mongoose;
+require('dotenv').config();
+
+// Write some dummy data into the database
+async function bootstrap() {
+    connectionStr = `mongodb://${process.env.MONGO_INITDB_ROOT_USERNAME}:${process.env.MONGO_INITDB_ROOT_PASSWORD}@${process.env.MONGO_HOST}:${process.env.MONGO_PORT}/admin`;
+    console.log(connectionStr);
+    await mongoose.connect(connectionStr);
+
+    const NftWalletSchema = new Schema({
+    address: String,
+    tokens: [{
+        collectionId: Number,
+        seriesId: Number,
+        tokenId: Number,
+    }]
+    });
+
+    const NftWallet = mongoose.model('NftWallet', NftWalletSchema);
+
+    // Create an instance of model SomeModel
+    const nftWallet = new NftWallet({
+        address: '5FWizEtxJTb2wPjWEqtEDetYTjgmWRmUNvRpMBh6ZDX4JJCt',
+        tokens: [
+            {
+                collectionId: 1,
+                SeriesId: 1,
```

Comment:
is this not case sensitive... the schema has it as seriesId and here we can use it as SeriesId?